code/modules/security levels/security levels.dm

```diff
@@ -33,6 +36,10 @@
 					security_announcement_down.Announce("[config.alert_desc_blue_downto]", "Attention! Security level lowered to blue.")
 				security_level = SEC_LEVEL_BLUE
 				SSnightlight.end_temp_disable()
+			if(SEC_LEVEL_YELLOW)
+				security_announcement_up.Announce("[config.alert_desc_yellow_to]", "Attention! Biohazard status declared!")

```

Comment:
I feel like this should use the standard `elevated to`/`lowered to` that the other levels use, with an additional blurb about the biohazard. Additionally, there's no announcement for lowering the alert, it seems?

Reply:
Going on yellow isn't raising or lowering the code level in this case as it's a biohazard level. The other codes gauge human threats while this deals with biological threats that aren't human.

Reply:
"Biohazard status declared" doesn't sound right. `Attention! Biohazard alert declared!` might be better.